### PR TITLE
Can 570 social login

### PR DESCRIPTION
**Pull request by ks-rahul**
the issue with email is not registered on a social account.

NOTE:- this is not tested test once deploy

### CODE REVIEW

File: src/components/ComponentPages/Registration/UI/email.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          type="link"
+          className={styles.close_btn}
+          onClick={closeModal}
+          icon={<CloseCircleOutlined />}
+        />
+      )}
+      <div className={styles.section_one}>
+        <Text type="danger" className={styles.otpNote}>
+          {isOTP
+            ? `Note : Verification code has been sent to your registered email address.`
```
Comment (akuks) [inline src/components/ComponentPages/Registration/UI/email.tsx]: Message must be come from the common message file

File: src/components/ComponentPages/Registration/UI/email.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            : `Note : Your email address is not returned from social account. You
```
Comment (akuks) [inline src/components/ComponentPages/Registration/UI/email.tsx]: Message must be come from the common message file

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/Registration/UI/email.tsx (hunk 1 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Registration/UI/email.tsx]: done

File: src/components/ComponentPages/Registration/UI/email.tsx (hunk 2 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Registration/UI/email.tsx]: done

File: src/messages/label.ts [hunk 3]
```diff
@@ -62,4 +62,8 @@ export const labels = {
   City: "City",
   Country: "country",
   listOfSupportedCamps: "List of supported camps",
+  otpLabel:
+    "Note : Verification code has been sent to your registered email address",
```
Comment (akuks) [inline src/messages/label.ts:66]: All the mesages must come from the common message file.

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/messages/label.ts (hunk 3 above)
Comment (ks-rahul) [reply src/messages/label.ts:66]: This message comes from the shared file.

**Pull request merged**